wildfly-builder-image/image.yaml

```diff
@@ -43,14 +43,16 @@ modules:
           - name: openjdk
             git:
               url: https://github.com/jboss-container-images/openjdk
-              ref: ubi8
+              ref: ubi9
           - name: wildfly-cekit-modules
             git:
               url: https://github.com/wildfly/wildfly-cekit-modules
               ref: main
           - name: wildfly-modules
             path: ../wildfly-modules
       install:
+          - name: jboss.container.user
```

Comment:
what is this new module for? Is it related to the switch to UBI 9?

Reply:
Yes, openjdk image, in their ubi9 branch renamed the user to default. We are using jboss.